packages/pn-personafisica-webapp/src/pages/Contacts.page.tsx

```diff
@@ -108,7 +109,11 @@ const Contacts = () => {
           apiId={CONTACT_ACTIONS.GET_DIGITAL_ADDRESSES}
           reloadAction={fetchAddresses}
         >
-          <Stack direction="column" spacing={4} mt={4}>
+          <ContactsSummaryCards
+            legalAddresses={legalAddresses}
```

Comment:
when you'll merge with feature branch you won't have legalAddresses or courtesyAddresses anymore. use redux selectors to get them inside the ContactsSummaryCards component